src/Telemetry/AzureDevOpsClient/AzureDevOpsClientFactory.cs

```diff
@@ -10,10 +11,12 @@ namespace Microsoft.DotNet.Internal.AzureDevOps
     public sealed class AzureDevOpsClientFactory : IAzureDevOpsClientFactory
     {
         private readonly IHttpClientFactory _httpClientFactory;
+        private readonly ExponentialRetry _exponentialRetry;
 
-        public AzureDevOpsClientFactory(IHttpClientFactory httpClientFactory) 
+        public AzureDevOpsClientFactory(IHttpClientFactory httpClientFactory, ExponentialRetry exponentialRetry) 
```

Comment:
This is such a... weird type... it's not doing anything other than just doing `services.AddTransient<AzureDevOpsClientFactory, IAzureDevOpsClientFactory>()`.  I don't think it's your fault, but it is a weird type that you are having to deal with each time.

I wonder if we could inject a "Func<IAzureDevOpsClient>" instead...